nservicebus/upgrades/asp-6to1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Azure Storage Persistence Upgrade Version 6 to 1
+summary: Instructions on how to upgrade NServiceBus.Azure Storage Persistence Version 6 to NServiceBus.Persistence.AzureStorage Version 1.
+tags:
+ - upgrade
+ - migration
+related:
+- nservicebus/azure-storage-persistence
+- nservicebus/upgrades/asp-saga-deduplication
+---
+
+## Saga de-duplication patch
```

Comment:
Can/Should this be a warning?

I would expect this page to open with the NuGet breakout so this seemed out of place.